sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashJoin.scala

```diff
@@ -39,7 +39,18 @@ private[joins] case class HashedRelationInfo(
     relationTerm: String,
     keyIsUnique: Boolean,
     isEmpty: Boolean)
-
+private[joins] case class HashJoinParams(
```

Comment:
Personally, I like observe one empty line between class declaration.

Reply:
+1

Usually we have blank line between class/trait definition.